packages/edge-demo-site/README.md

```diff
@@ -0,0 +1,54 @@
+# Welcome to Remix!
+
+- [Remix Docs](https://remix.run/docs)
+- [Netlify Edge Functions Overview](https://docs.netlify.com/netlify-labs/experimental-features/edge-functions)
+
+## Netlify Setup
+
+1. Install the [Netlify CLI](https://www.netlify.com/products/dev/):
+
+```sh
+npm i -g netlify-cli
+```
+
+If you have previously installed the Netlify CLI, you should update it to the latest version:
+
+```sh
+npm i -g netlify-cli@latest
+```
+
+2. Sign up and log in to Netlify:
+
+```sh
+netlify login
+```
+
+3. Create a new site:
+
+```sh
+netlify init
+```
+
+## Development
+
+The Netlify CLI starts your app in development mode, rebuilding assets on file changes.
+
+```sh
+npm run dev
+```
+
+Open up [http://localhost:3000](http://localhost:3000), and you should be ready to go!
+
+## Deployment
+
+There are two ways to deploy your app to Netlify, you can either link your app to your git repo and have it auto deploy
+changes to Netlify, or you can deploy your app manually. If you've followed the setup instructions already, all you need
```

Comment:
```suggestion
changes to Netlify, or you can deploy your app manually. If you've followed the setup instructions already, run:
```
[sand] Option to shorten this for clarity.

Reply:
I think this is an older version. The current text is this. Not sure why that wasn't appearing, but GitHub is having some issues with commit suggestions at the moment, so maybe that's why you didn't see my latest commits. 🤔 

```
# Remix Demo Site for Netlify Edge Functions

This is a demo site for [Remix](https://remix.run) apps running on
[Netlify Edge Functions](https://docs.netlify.com/edge-functions/overview/).

It is used to test the @netlify/remix-edge-adapter and @netlify/remix-runtime packages. It is not intended to be used as
a template for Remix apps.

```